backend/app/api/endpoints/graph.py

```diff
@@ -112,15 +186,30 @@ async def getMatchRuns(meta: dict):
                     f'benchmark: "{meta["benchmark"]}"'
                     f' AND workerNodesType: "{meta["workerNodesType"]}"'
                     f' AND masterNodesType: "{meta["masterNodesType"]}"'
-                    f' AND masterNodesCount: "{meta["masterNodesCount"]}"'
-                    f' AND workerNodesCount: "{meta["workerNodesCount"]}"'
                     f' AND platform: "{meta["platform"]}"'
                     f' AND ocpVersion: {version}*'
                     f' AND jobStatus: success'
                     )
             }
         }
     }
+    if workerCount :
```

Comment:
should the above lines of setting query and this if statement be in an if/else?

Reply:
I understood it as, this is the default query if we get nodeCount data, replace with this query

Reply:
Understood but why set it to begin with if we know we are going to overwrite. Just preference, don't need to hold up merging

Reply:
yeah... it only changes for kube-burner queries -- to include the number of workers and masters...